docs/OV_Runtime_UG/Integrate_with_customer_application.md

```diff
@@ -66,235 +61,124 @@ Use the following code to create Inference Engine Core to manage available devic
 > conversion will be invoked automatically using color format configured on step 3.
 >
-> * `GetBlob()` logic is the same for pre-processable and not pre-processable input. Even if it is
+> * `get_tensor()` logic is the same for pre-processable and not pre-processable input. Even if it is
 > called with input configured as resizable or as having specific color format, a blob allocated by
 > an infer request is returned. Its size and color format are already consistent with the
 > corresponding values of the read network. No pre-processing will happen for this blob. If you
-> call `GetBlob()` after `SetBlob()`, you will get the blob you set in `SetBlob()`.
+> call `get_tensor()` after `set_tensor()`, you will get the blob you set in `set_tensor()`.
 
 #### Step 6. Start Inference
 
 Start inference in asynchronous or synchronous mode. Async API usage can improve overall frame-rate of the application, because rather than wait for inference to complete, the app can continue doing things on the host, while accelerator is busy.
 
 * For synchronous inference request:
-   ```cpp
-   infer_request.Infer();
-   ```
+   @snippet snippets/src/main.cpp part11
 
 * For asynchronous inference request: 
-  ```cpp
-  infer_request.StartAsync();
-  infer_request.Wait(InferenceEngine::InferRequest::WaitMode::RESULT_READY);
-  ```
-  `StartAsync` returns immediately and starts inference without blocking main thread, `Infer` blocks main thread and returns when inference is completed. Call `Wait` for waiting result to become available for asynchronous request.
+   @snippet snippets/src/main.cpp part12
+  `start_async` returns immediately and starts inference without blocking main thread, `infer` blocks main thread and returns when inference is completed. Call `wait` for waiting result to become available for asynchronous request.
 
   There are three ways to use it:
-      * specify maximum duration in milliseconds to block for. The method is blocked until the specified timeout has elapsed, or the result becomes available, whichever comes first.
-      * `InferenceEngine::InferRequest::WaitMode::RESULT_READY` - waits until inference result becomes available
-      * `InferenceEngine::InferRequest::WaitMode::STATUS_ONLY` - immediately returns request status.It does not
-      block or interrupts current thread.
-   
+      * `ov::InferRequest::wait_for()` - specify maximum duration in milliseconds to block for. The method is blocked until the specified timeout has elapsed, or the result becomes available, whichever comes first.
+      * `ov::InferRequest::wait()` - waits until inference result becomes available
+
 
 Both requests are thread-safe: can be called from different threads without fearing corruption and failures.
 
-Multiple requests for single `ExecutableNetwork` are executed sequentially one by one in FIFO order.
+Multiple requests for single `CompiledModel` are executed sequentially one by one in FIFO order.
```

Comment:
@myshevts  do we need to refer to guide with streams / performance? Because we allow to run requests in parallel with streams, even w/o streams - some stages of multiple inference requests can be simultaneously executed